### ISSUE TITLE
Getting error while adding and removing data in cluster

### ISSUE THREAD

**Issue by pravindot17** [arjunmehta/node-georedis#34]
Hi,

It's a really interesting and great library, thanks for saving lot of our work.

I am getting below error when I run `nearby` in redis cluster.

> {"correlationId":"89a9922e-019d-47d5-a8fd-804ae261b3b0","data":{"message":"MOVED 3400 10.20.41.108:6379","statusCode":500}}
What could be the reason, does it support cluster?

**Comment by arjunmehta** [comment]
@pravindot17 Sorry for not seeing this until now.

Which redis library are you using? Are you using `node-redis`?

**Comment by pravindot17** [comment]
@arjunmehta Yes, I am using [ioredis](https://www.npmjs.com/package/ioredis).
